<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>도서관리 프로그램 v0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>78</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>관리번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leReg">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>80</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>112</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>도서명</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>114</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePub">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>148</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>146</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>출판사</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKind">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>182</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>180</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>도서종류</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEtc2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>216</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>215</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>도서내용</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEtc3">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>250</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>249</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>비고</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDate">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>등록일</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAuthor">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>저자</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEtc1">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>주제분류</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>69</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>98</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>127</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
